packages/terra-tag/tests/jest/Tag.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Tag from '../../src/Tag';
+
+describe('Tag', () => {
+  const defaultRender = <Tag text="default" />;
+
+  // Snapshot Tests
+  it('should render a tag', () => {
+    const tag = shallow(<Tag text="Normal" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  it('should render as an anchor when provided an href', () => {
+    const tag = shallow(<Tag text="href" href="MockHref" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  it('should render an icon and a text', () => {
+    const testElement = <img alt="icon" />;
+    const tag = shallow(<Tag icon={testElement} text="text" />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  // Prop Tests
+  it('should render an interactive tag', () => {
+    const tag = shallow(<Tag text="text" onClick={() => {}} />);
+    expect(tag).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class tag', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper.prop('className')).toContain('tag');
```

Comment:
How about these structure tests? Can they be changed to snapshots as well? I see the snapshots are doing different elements such as span and button, and also setting the class attributes as well.